protected/humhub/modules/admin/controllers/GroupController.php

```diff
@@ -276,13 +278,25 @@ public function actionAdminUserSearch($keyword, $id)
 
     public function checkGroupAccess($group)
     {
-        if(!$group) {
+        if (!$group) {
             throw new HttpException(404, Yii::t('AdminModule.user', 'Group not found!'));
         }
 
-        if($group->is_admin_group && !Yii::$app->user->isAdmin()) {
+        if ($group->is_admin_group && !Yii::$app->user->isAdmin()) {
             throw new HttpException(403);
         }
     }
 
+    public function actionReassignAll($id)
+    {
+        $job = new ReassignAllDefaultSpaces();
+        if (QueueHelper::isQueued($job)) {
+            return $this->render('reassign-already-running', ['groupId'=>$id]);
```

Comment:
A extra view should not be necessary here. 

It should be sufficient to work with status messages here. e.g.
- The Space memberships of this group will be reassigned. This process can take up to several minutes.

Example:
https://github.com/humhub/humhub/blob/master/protected/humhub/modules/admin/controllers/InformationController.php#L115

Reply:
Done!